tests/XrdOucTests/XrdOucUtilsTests.cc

```diff
@@ -103,7 +103,7 @@ static const std::string authz_strings[] = {
   "/path/test.txt?scitag.flow=44&authz=REDACTED done close.",
   "/path/test.txt?authz=REDACTED&scitag.flow=44 done close.",
   "(message: kXR_stat (path: /tmp/xrootd/public/foo?authz=REDACTED&pelican.timeout=3s, flags: none) ).",
-  "(message: kXR_stat (path: /tmp/xrootd/public/foo?pelican.timeout=3s&authz=REDACTED, flags: none) ).",
+  "(message: kXR_stat (path: /tmp/xrootd/public/foo?pelican.timeout=3s&authz=REDACTED flags: none) ).",
```

Comment:
This is a consequence of the unrelated change before. The test is correct and cannot be changed. Please remove this modification from the pull request.